pkg/R/deserialize.R

```diff
@@ -26,10 +26,12 @@ readTypedObject <- function(con, type) {
     "b" = readBoolean(con),
     "d" = readDouble(con),
     "r" = readRaw(con),
+    "D" = readDate(con),
```

Comment:
Can you update the type mapping at the top of this file ? Its good to keep code comments up to date. Similar updates in serialize.R and SerDe.scala